{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2019-4119",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "http://www.ibm.com/support/docview.wss?uid=ibm10878460",
          "name": "http://www.ibm.com/support/docview.wss?uid=ibm10878460",
          "refsource": "CONFIRM",
          "tags": [

          ]
        },
        {
          "url": "https://exchange.xforce.ibmcloud.com/vulnerabilities/158145",
          "name": "ibm-cloud-cve20194119-proxy (158145)",
          "refsource": "XF",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "IBM Cloud Private Kubernetes API server 2.1.0, 3.1.0, 3.1.1, and 3.1.2 can be used as an HTTP proxy to not only cluster internal but also external target IP addresses. IBM X-Force ID: 158145."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2019-05-17T16:29Z",
  "lastModifiedDate": "2019-05-17T16:29Z"
}